nodejs/week2/warmup/src/backend/routes/mul.js

```diff
@@ -0,0 +1,7 @@
+const mul = function (req, res) {
```

Comment:
I would suggest don't use very short names for variables. It is better to write multiply.